01-node-tutorial/answers/content/big.txt

```diff
@@ -0,0 +1,10000 @@
+hello world 0
```

Comment:
Not a big deal, but I think you'll want to remove this file so it's not tracked in git. You'll notice the `01-node-tutorial/.gitignore` file has this filename listed. Since you created it in the `answers` folder, it didn't get caught by the gitignore.

Reply:
I c, thanks. I've deleted big.txt from Github just in case :)